docs/pages/router/reference/troubleshooting.mdx

```diff
@@ -44,6 +44,16 @@ export function App() {
 registerRootComponent(App);
 ```
 
+You will need to also update your app's main entry point:
+
+```json package.json
+{
+  ...
+  "main": "index.js"
+  ... 
```

Comment:
```suggestion
{
  /* @hide ... */ /* @end */
  "main": "index.js"
```

For consistency in code blocks, we prefer using the annotation to indicate nothing else changes in the file as above.